L1Trigger/L1TGlobal/src/GlobalBoard.cc

```diff
@@ -342,6 +346,21 @@ void l1t::GlobalBoard::receiveCaloObjectData(const edm::Event& iEvent,
       }  //end loop over Bx
     }
   }
+  if (receiveCICADA) {
+    edm::Handle<BXVector<float>> cicadaScoreHandle;
+    iEvent.getByToken(CICADAInputToken, cicadaScoreHandle);
+    if (not cicadaScoreHandle.isValid()) {
+      if (m_verbosity) {
+        edm::LogWarning("L1Tglobal") << "\nWarning: Input tag for the CICADA score"
+                                     << "\nrequested in configuration, but not found in the event.\n"
+                                     << "\nSetting score to 0.0";
+        setCICADAScore(0.0);
```

Comment:
This setting to zero should be outside of the if block printingthe message.

Reply:
Thank you, good catch.